src/MongoDB.Driver.Core/Core/Servers/ServerMonitor.cs

```diff
@@ -89,6 +133,7 @@ public void Initialize()
             if (_state.TryChange(State.Initial, State.Open))
             {
                 MonitorServerAsync().ConfigureAwait(false);
+                _roundTripTimeMonitor.RunAsync().ConfigureAwait(false);
```

Comment:
Should we use a thread instead of a Task here? Perhaps this might help us avoid the issues we've seen in the past when applications create too many `Task`s?

Reply:
I guess there is a ticket about dedicated threads for serverMonitor, but I guess it's a bit out of the scope for this ticket.

Reply:
Fair enough!

Reply:
Second question: would it be helpful at all to store  the `Task` that `_roundTripTimeMonitor` returns? 
Nit: If we totally don't care about the `Task` that's returned, should we use the discard variable?